Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -147,13 +165,25 @@ private void InteractionManager_SourceLost(InteractionSourceState state)
 
         private void GestureRecognizer_TappedEvent(InteractionSourceKind source, int tapCount, Ray headRay)
         {
-            OnTap();
+            ProcessTap();
```

Comment:
I'm not sure exactly what tapCount represents. Is it the cumulative taps or a period of time?